SharedPackages/BrowserServicesKit/Sources/BrowserServicesKit/SecureVault/SecureVaultModels.swift

```diff
@@ -203,9 +203,8 @@ public struct SecureVaultModels {
             return String(data: cardNumberData, encoding: .utf8)!
         }
 
-        public var displayName: String {
-            let type = CreditCardValidation.type(for: cardNumber)
-            return "\(type.displayName) (\(cardSuffix))"
+        public var displayNumber: String {
```

Comment:
This also impacts the Password Management UI, not sure if that was intentional? 🙂  
<img width="500" alt="image" src="https://github.com/user-attachments/assets/03462e44-093f-4be9-a076-80df18819cb2" />